backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -62,4 +67,13 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 	public Map<UUID, Shop> getAllShops() {
 		return shopMap;
 	}
+
+	public List<Token> getTokens(UUID shopId) {
+		if (shopMap.get(shopId).getStatus() == ShopStatus.ACTIVE) {
+			return shopIdToListOfTokensMap.get(shopId).stream().filter(token -> token.getStatus() == Status.ACTIVE)
+					.collect(Collectors.toList());
+		}
+		Logger.getLogger(InMemoryVirtualQueueRepository.class.getName()).log(Level.SEVERE, "Tried to fetch tokens of a shop which is not in the ACTIVE state.");
```

Comment:
Add a comment above this line.
"TODO(b/): Simplify logging across all backend APIs"

Generally we follow this pattern: "TODO(b/#some bug number here): Do this thing..."

Reply:
Let's get our base code checked in, we can later get back to logging, etc.

Reply:
Added Todo.